pulser/simulation/simulation.py

```diff
@@ -98,6 +98,15 @@ def __init__(
         self._qdict = self._seq.qubit_info
         self._size = len(self._qdict)
         self._tot_duration = self._seq.get_duration()
+
+        # Type hints for attributes defined outside of __init__
```

Comment:
It seems this comment is not accurate since these attributes _are_ defined inside `__init__`
Also see https://github.com/pasqal-io/Pulser/issues/335

Reply:
In what? They are declared here, but they are defined in other methods. Do you mean that these methods are run inside of `__init__`?

Reply:
I understand now, I thought it meant they were defined in the class, never mind.